go.mod

```diff
@@ -2,29 +2,31 @@ module github.com/iden3/go-iden3-auth/v2
 
 go 1.18
 
+// replace github.com/iden3/iden3comm/v2 => ../iden3comm
+
 require (
-	github.com/btcsuite/btcd/btcutil v1.1.3
+	github.com/btcsuite/btcd/btcutil v1.1.6
 	github.com/ethereum/go-ethereum v1.11.5
 	github.com/golang/mock v1.6.0
 	github.com/google/uuid v1.6.0
 	github.com/iden3/contracts-abi/state/go/abi v1.0.1
-	github.com/iden3/go-circuits/v2 v2.3.0
-	github.com/iden3/go-iden3-core/v2 v2.2.0
-	github.com/iden3/go-iden3-crypto v0.0.16
-	github.com/iden3/go-jwz/v2 v2.1.1
+	github.com/iden3/go-circuits/v2 v2.4.0
+	github.com/iden3/go-iden3-core/v2 v2.3.1
+	github.com/iden3/go-iden3-crypto v0.0.17
+	github.com/iden3/go-jwz/v2 v2.2.0
 	github.com/iden3/go-rapidsnark/types v0.0.3
 	github.com/iden3/go-rapidsnark/verifier v0.0.5
-	github.com/iden3/go-schema-processor/v2 v2.4.0
-	github.com/iden3/iden3comm/v2 v2.5.1
+	github.com/iden3/go-schema-processor/v2 v2.5.0
+	github.com/iden3/iden3comm/v2 v2.7.1-0.20241015160531-d8ea65210731
```

Comment:
It needs code in https://github.com/iden3/iden3comm/pull/60 that should be accepted, merged and released before accepting this one.